CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.7.3 (2020-01-24)
```

Comment:
I think we ned to update the date to the current date before we merge.

Reply:
Done.